deps/rabbit/src/rabbit_quorum_queue.erl

```diff
@@ -1672,34 +1672,36 @@ online(Q) when ?is_amqqueue(Q) ->
                   is_pid(Pid)].
 
 format(Q, Ctx) when ?is_amqqueue(Q) ->
```

Comment:
this function is called for every queue when using the /queues HTTP API, depending on sort columns this could mean _all_ queues in the system. Please compare HTTP API query times with and without this change with 5k quorum queues in a 3 node cluster.

I think we need to find another way of achieving this. Perhaps we need to persist the voting set in the queue record queue type state